src/containers/Resources/Resources.jsx

```diff
@@ -63,36 +50,31 @@ class Resources extends Component {
     const {
       match: { params },
       t,
-      title,
+      topic,
       resourceTypes,
-      supplementaryResources,
-      coreResources,
       location,
       locale,
       ndlaFilm,
     } = this.props;
 
     if (
-      coreResources &&
-      coreResources.length === 0 &&
-      supplementaryResources.length === 0
+      topic?.coreResources?.length === 0 &&
+      topic?.supplementaryResources.length === 0
     ) {
       return null;
     }
 
-    const subjectTopicPath = getSubjectTopicPath(params);
-
     const resourceToLinkProps = resource =>
       resourceToLinkPropsHelper(
         resource,
-        subjectTopicPath,
+        topic.path,
```

Comment:
Sender heller inn topic-objekt for då kan vi berre bruke pathen til topicen for å utlede lenker.